api-extractor.json

```diff
@@ -173,7 +173,7 @@
      * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
      * DEFAULT VALUE: "<projectFolder>/temp/<unscopedPackageName>.api.json"
      */
-    // "apiJsonFilePath": "<projectFolder>/temp/<unscopedPackageName>.api.json"
+    "apiJsonFilePath": "<projectFolder>/build/temp/<unscopedPackageName>.api.json"
```

Comment:
@fhinkel to keep the git index clean while testing locally, I found it helpful to just stash this stuff in `build`.  Maybe worth considering broadly for our scripts.